tests/test_func.py

```diff
@@ -120,20 +121,29 @@ def gen_tests(filter_rgx):
         base = module_file.replace(".py", "").split("_")[1]
         dependencies = _get_tests_info(INPUT_DIR, MSG_DIR, base, ".py")
         tests.append((module_file, messages_file, dependencies))
-
     if UPDATE:
         return tests
-
     assert len(tests) < 196, "Please do not add new test cases here."
     return tests
 
 
+TEST_WITH_EXPECTED_DEPRECATION = ["func_excess_escapes.py"]
+
+
 @pytest.mark.parametrize(
     "module_file,messages_file,dependencies",
     gen_tests(FILTER_RGX),
     ids=[o[0] for o in gen_tests(FILTER_RGX)],
 )
 def test_functionality(module_file, messages_file, dependencies):
+    if module_file in TEST_WITH_EXPECTED_DEPRECATION:
+        # Remove <unknown>:x: DeprecationWarning: invalid escape sequence
```

Comment:
I do not understand this comment. This modif doesn't remove anything but ensure a `Deprecated` warning is emitted? Am i wrong?

Reply:
Yes, it probably should be "Catch  <unknown>:x: DeprecationWarning: invalid escape sequence so it's not shown during tests", what do you think ?

Reply:
Ok let's do that! :wink: